examples/charts/steps.py

```diff
@@ -1,9 +1,6 @@
 from collections import OrderedDict
 
-import numpy as np
-import pandas as pd
-from bokeh.charts import Chart, Step, Line
-from bokeh.plotting import show,output_file
+from bokeh.charts import Step, show, output_file
```

Comment:
I guess we can wait for our users to choose... if we see they use the `show` in each API, then we can leave that for them... but we have to choose a way for our examples, and I guess using the `state` is the correct one...